proto/process/agent.proto

```diff
@@ -122,10 +118,10 @@ message CollectorECSTask {
 	int32 groupId = 5;
 	int32 groupSize = 6;
 	repeated ECSTask tasks = 7;
-	ECSAgentInfo agentInfo = 8;
-	repeated string tags = 9;
-	Host host = 10; // Post-resolved field
-	string hostName = 11;
+	repeated string tags = 8;
```

Comment:
without more context, I think we need to keep existing indexes and not break the wire format